lib/ansible/module_utils/redfish_utils.py

```diff
@@ -907,7 +927,16 @@ def get_nic_inventory(self, resource_type):
         result["entries"] = nic_results
         return result
 
-    def get_psu_inventory(self):
+    def get_multi_nic_inventory(self, resource_type):
+        ret = True
+        entries = []
+        for systems_uri in self.systems_uris:
+            inventory = self.get_nic_inventory(resource_type, systems_uri)
+            ret = inventory.pop('ret') and ret
+            entries.append(inventory['entries'])
```

Comment:
To match the code in aggregate():
```suggestion
            if 'entries' in inventory:
                entries.append(({'systems_uri': systems_uri},
                                inventory['entries']))
```